00-Getting-Started/04-Other-Important-Info.md

```diff
@@ -27,7 +27,7 @@ Then, you use a search query like the following:
 | Find real-world examples of code that uses libraries (e.g. `aff-bus`) | [`language:purescript path:src/ in:file "Effect.Aff.Bus"`](https://github.com/search?q=language%3Apurescript+path%3Asrc%2F+in%3Afile+Bus) | Search a project's `src` directory for files whose content mentions the `Effect.Aff.Bus` module at some point. (Note: the full module name must be surrounded by quotes)
 | Find real-world examples of code that uses testing libraries (e.g. `aff-bus`) | [`language:purescript path:test/ in:file "Test.QuickCheck"`](https://github.com/search?q=language%3Apurescript+path%3Atest%2F+in%3Afile+QuickCheck) | Search a project's `test` directory for files whose content mentions the `Test.QuickCheck` module at some point. (Note: the full module name must be surrounded by quotes)
 
-## Documenation
+## Documentation
```

Comment:
Man... My fingers have muscle memory in typing typos!
Thanks for the fix!

Reply:
I think that these kind of small word typos aren't so important that they deserve to be always fixed. But if they're along something else I think it goes very well to catch them up. That was a good starter to add an useful improvement to this document.

I'm just thinking here that if somebody lands to the preface directly, he won't read the README.md. Maybe we should fix that as well with something.

Reply:
> If somebody lands to the preface directly, he won't read the README.md. Maybe we should fix that as well with something. Do you have ideas on that? I'd guess I would just throw a link somewhere, maybe forming a third paragraph before the "Intended audience", and then eventually strive to sync them.

Which readme are you referring to? This project's readme? Or the site repo's readme?

Reply:
This readme: https://github.com/JordanMartinez/purescript-jordans-reference/blob/latestRelease/Readme.md

I wonder if you have an another readme that I am not aware about now?

Reply:
So, I'm writing the content for the work in this repo (i.e. purescript-jordans-reference). However, I'm hosting the website version of the repo's content as an `mdbook` via https://github.com/jordanmartinez/purescript-jordans-reference-site (notice the `-site` suffix). That also has a Readme file, so I wasn't sure whether you were talking about that one.

Anyway, this repo's readme is designed for people who come across this repo itself. The Preface in the website version of the work is designed for people who come across that work via the website link. These two things are intentionally separate on purpose. 

However, it sounds like you are expecting parts of this repo's readme to appear in the website version's Preface. What content do you think should appear there? If it's the content you just wrote, you would need to submit a PR to the `-site` repo because that's currently how my deployment workflow works.

Reply:
Now I see how you've arranged this. I completely missed the -site repository.

Ok, I'll drop it in there later.